hooks/001-extra-packages.chroot

```diff
@@ -244,6 +244,12 @@ case "$(dpkg --print-architecture)" in
         ;;
 esac
 
+case "$(dpkg --print-architecture)" in
+    arm64|armhf)
+        PACKAGES+=(tee-supplicant)
```

Comment:
Maybe one day, we should make a list of all packages that are from universe rather than main.